template_content_swapper/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,6 @@
+This module provides a generic functionality to replace QWeb view elements.
+
+Examples:
+
+* Replace 'Salesperson' label with 'Sales Representative' in the quotation print.
+* Replace 'Add to Cart' button with 'Add to Basket' in the eCommerce product page.
```

Comment:
Hi, normally replacing elements is carried out via a child template with an XPATH replacement.

Could you elaborate in this description text the use cases where using your module will be better?